src/Nacos/NacosClient.php

```diff
@@ -20,6 +20,7 @@ class NacosClient
     const DEFAULT_TIMEOUT = 3;
 
     const DEFAULT_GROUP = 'DEFAULT_GROUP';
+    const DEFAULT_NAMESPACEID = 'public';
```

Comment:
namespace id 中间推荐加个下划线
```suggestion
    const DEFAULT_NAMESPACE_ID = 'public';
```